03.asynchronous/callback/no_error.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import sqlite3 from "sqlite3";
+const db = new sqlite3.Database(":memory:");
+
+const createTableQuery = `
+  CREATE TABLE IF NOT EXISTS books (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    title TEXT NOT NULL UNIQUE
+  )
+`;
+
+db.run("drop table if exists books", (err) => {
+  if (err) throw err;
+  db.run(createTableQuery, (err) => {
+    if (err) throw err;
+    db.run("INSERT INTO books(title) values(?)", "test", (err) => {
```

Comment:
`CREATE TABLE` のクエリとスタイルを合わせるならカッコの前にスペースを入れましょう。それから SQL のキーワード部分は大文字で書くようにしましょう。また、`books.title` が `"test"` のデータを挿入しているようですが、`books.title` として意味を持つデータを入れたほうがよいと思います。

```suggestion
    db.run("INSERT INTO books (title) values (?)", "test", (err) => {
```

Reply:
キーワードを大文字にし、空白を入れました。
test→Ruby入門に変更しました。